blark/transform.py

```diff
@@ -614,30 +614,43 @@ class Variable(Expression):
     ...
 
 
+@dataclass
 @_rule_handler(
     "indirection_type",
     "pointer_type",
 )
-class IndirectionType(Enum):
+class IndirectionType(Literal):
```

Comment:
I think `Literal` might be the wrong base class here - it's reserved for more like integers, strings, byte strings, and such.

What would be the right one? Hmm, actually, I think it shouldn't inherit from anything (i.e., `class IndirectionType:`)

Reply:
Definitely! I hadn't even thought of this!